python/lsst/meas/algorithms/loadReferenceObjects.py

```diff
@@ -1076,6 +1103,14 @@ def addAliasesForOneFilter(filterName, refFilterName):
                 camFluxErrName = camFluxName + "Err"
                 aliasMap.set(camFluxErrName, refFluxErrName)
 
+        if self.config.anyFilterMapsToThis is not None:
+            refFluxName = self.config.anyFilterMapsToThis + "_flux"
+            if refFluxName not in schema:
+                msg = f"Unknown reference filter for anyFilterMapsToThis='{refFluxName}'"
+                raise RuntimeError(msg)
+            aliasMap.set("anyFilterMapsToThis", refFluxName)
+            return  # this is mutually exclusive with filterMap
```

Comment:
Please consider moving this code block before the definition of addAliasesForOneFilter, since this code block doesn't use that function. I found it confusing to read through a function definition that was not used.